store/src/server/mod.rs

```diff
@@ -12,23 +12,20 @@ mod api;
 // STORE INITIALIZER
 // ================================================================================================
 
-#[instrument(skip(config, db))]
+#[instrument(target = "miden-store", skip_all)]
 pub async fn serve(
     config: StoreConfig,
     db: Db,
 ) -> Result<()> {
+    info!(target: COMPONENT, ?config);
```

Comment:
I would probably make this similar to line 28 below. For example, something like:
```Rust
info!(target: COMPONENT, host = config.endpoint.host, port = config.endpoint.port, "Starting server");
```
Or if we implement `Display` on `config`, we could probably do:
```Rust
info!(target: COMPONENT, %config, "Starting server");
```
But either way, we should do it consistently here and on line 28.